### DISCUSSION TITLE
Opening of new tracks

### DISCUSSION THREAD

**Discussion by Giovani93** [audacity/audacity#7544] [Q&A]
### Bug description

Hello.
When I've opened new audio and I've pressed play, It doesn't play from the begining at first time. After second time It played normally from the begining.
I use newest version of Audacity and Windows 10. This issue only appears, when for example new window of Audacity is opened, or application is opened.

### Steps to reproduce

Step 1: Opening audio
Step 2: Pressing play

### Expected behavior

Audio is played, but I don't listen begining of this audio.

### Actual behavior

Nothing, I am blind. I coudln't describe It.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

### Additional context

It can happened after opening new window of Audacity, or just application of Audacity and do things described on steps.

**Comment by DavidBailes** [comment]
I can reproduce this issue on Windows 11. The issue seems to be worse when using the hosts MME or direct sound, rather than WASAPI.

Alternative steps to reproduce:
1. First open Audacity, and check the host. If set to WASAPI, set it to MME or direct sound, and then close Audacity.
2. Open Audacity.
3. Generate a tone 200ms long.
4. Press Spacebar to play.

Expected behaviour:
The tone is played.

Actual behaviour:
Silence.

Note: If you press spacebar again, the tone is then played.
